src/cljs/chronograph_web/events.cljs

```diff
@@ -1,25 +1,5 @@
 (ns chronograph-web.events
```

Comment:
Is this being used anywhere?

Reply:
It's being used to force requiring the other events namespaces so that event handlers are registered.